modules/reindex/src/main/java/org/elasticsearch/index/reindex/AbstractAsyncBulkByScrollAction.java

```diff
@@ -19,6 +19,18 @@
 
 package org.elasticsearch.index.reindex;
 
+import java.util.ArrayList;
```

Comment:
Can you configure your IDE to stop re-ordering imports? I think we already settled on the standard so we should stick with it so PRs don't flip-flop imports depending on who's working on them